packages/front-end/components/Settings/Team/InviteList.tsx

```diff
@@ -105,8 +109,13 @@ const InviteList: FC<{
     <div>
       <h5>Pending Invites{` (${invites.length})`}</h5>
       <div className="text-muted mb-2">
-        Invites that have been sent but have not yet been accepted.{" "}
-        <strong>Invited users count towards plan seat limits.</strong>
+        Invites that have been sent but have not yet been accepted.
+        {actualPlan !== "enterprise" && (
```

Comment:
I think you should keep this blurb for everyone.  All enterprise plans that I know of have a seat limit.  Up until recently though we haven't been enforcing it, and we have been relying on our enterprise customers to tell us when they go over, and we haven't even been letting them know that they are over in any obvious way.  But now we have a hardCap on some enterprise licenses we have been giving out, and there is a PR in review where we will let everyone know in the AccountPlanNotices in the top nav bar when they go over the limit whether they have a hard cap or not.  So I think this is useful information regardless.